README.md

```diff
@@ -106,6 +106,14 @@ rosdep install --from-paths src --ignore-src -r -y
 > poetry install --with openset,nomad
 > ```
 
+#### 1.4 Configure RAI
+
+Run the configuration tool to set up your vendor and other settings:
+
+```bash
+streamlit run src/rai/rai/utils/configurator.py
```

Comment:
`streamlit` is not visible without virtual env. But project is not built yes, so cannot use `setup_shell.sh`. So I think `poetry shell`.
```suggestion
```bash
poetry shell
streamlit run src/rai/rai/utils/configurator.py
```